docs/pages/versions/unversioned/sdk/splash-screen.mdx

```diff
@@ -80,6 +85,116 @@ export default function App() {
 }
 ```
 
+## Configuration
+
+### Expo config
+
+To configure `expo-splash-screen`, use the built-in [config plugin](/guides/config-plugins) in the Expo config (**app.json** or **app.config.js**) for [EAS Build](/build/introduction) or with `npx expo run:[android|ios]`. The plugin allows you to configure the following properties that cannot be set at runtime and require building a new app binary to take effect:
+
+<ConfigPluginProperties
+  properties={[
+    {
+      name: '<platform>.backgroundColor',
+      description: 'Color to fill the splash screen screen background.',
+    },
+    {
+      name: '<platform>.resizeMode',
+      default: 'contain',
+      description: 'Resize mode to render the splash screen image on a device, must be one of "cover", "contain" or "native".',
+    },
+    {
+      name: '<platform>.image',
+      description:
+        'Image to render on the splash screen, must be a png file. Both local paths or remote URLs are supported.',
+    },
+    {
+      name: '<platform>.dark.*',
+      description: 'Dark mode specific configuration for the splash screen, defined as nested object, with the same properties without the ".dark" prefix.'
+    },
+    {
+      name: 'ios.tabletImage',
+      platform: 'ios',
+      description:
+        'Image to render on the splash screen on tablets, must be a png file. Both local paths or remote URLs are supported.',
+    },
+    {
+      name: 'android.mdpi',
+      platform: 'android',
+      description: 'The natural sized image, or baseline, of the splash screen image, must be a .png file.',
+    },
+    {
+      name: 'android.hdpi',
+      platform: 'android',
+      description: 'The 1.5x scale of the sized image of the splash screen image, must be a .png file.',
+    },
+    {
+      name: 'android.xhdpi',
+      platform: 'android',
+      description: 'The 2x scale of the sized image of the splash screen image, must be a .png file.',
+    },
+    {
+      name: 'android.xxhdpi',
+      platform: 'android',
+      description: 'The 3x scale of the sized image of the splash screen image, must be a .png file.',
+    },
+    {
+      name: 'android.xxxhdpi',
+      platform: 'android',
+      description: 'The 4x scale of the sized image of the splash screen image, must be a .png file.',
+    },
+  ]}
+/>
+
+Here is an example of using the config plugin in the Expo config file:
+
+```json app.json
+{
+  "expo": {
+    "plugins": [
+      [
+        "expo-splash-screen",
```

Comment:
My only concern here would be if you used expo-splash-screen as a plugin and didn't define the standard values, would nothing show up in Expo Go?

Reply:
Yeah, if you only use a config plugin, Expo Go wouldn't be affected. I reused the standard plugin definition we have from notifications, but I see your point.

Maybe we could get away with just the XDL schema. That's populated on the "app.js / app.config.js" page, and described. But, visibility for the user might be lacking if you don't know where to look.